doc/source/license.rst

```diff
@@ -7,6 +7,11 @@ License
 
 OPEN SOURCE LICENSE AGREEMENT
 =============================
+BSD 3-Clause License
```

Comment:
Please check this, I think this has a CC license. We are not allowed to unilaterally change the license.

Reply:
OK I found the text buried in LICENSE.rst. I'll change this line to "Creative Commons License"

Reply:
do you know if it's the usual CC BY 4.0? I can't find this in the existing doc/source/license.rst or LICENSE.rst

Reply:
just reproduce here the license text from the LICENSE.rst   It predates CC BY 4.0 I think